_includes/footer_scripts.html

```diff
@@ -1,7 +1,9 @@
 <script src="{{ site.baseurl }}/assets/mobile-menu.js"></script>
 <script src="{{ site.baseurl }}/assets/scroll-to-anchor.js"></script>
 <script src="{{ site.baseurl }}/assets/external-links-new-tab.js"></script>
-<script src="{{ site.baseurl }}/assets/search-bar.js"></script>
+{% if page.deep-learning-index != true %}
+  <script src="{{ site.baseurl }}/assets/search-bar.js"></script>
+{% endif %}
```

Comment:
You'll also need to check for the thank you page. The console error still shows there